### PR TITLE
e2e: get logs for sidecar containers

### PR DESCRIPTION
**Pull request by aramase**
Signed-off-by: Anish Ramasekar <anish.ramasekar@gmail.com>

**What this PR does / why we need it**:
- Gets log for sidecar containers

**Which issue(s) this PR fixes** *(optional, using `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when the PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: test/bats/tests/azure/job_templates/kubernetes_windows.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
         "agentPoolProfiles": [
             {
                 "name": "windowspool1",
-                "count": 2,
+                "count": 1,
```
Comment (aramase) [inline test/bats/tests/azure/job_templates/kubernetes_windows.json:27]: Reducing to 1 node as it's easier to debug

File: test/bats/tests/azure/job_templates/kubernetes_windows.json [hunk 2]
```diff
@@ -51,5 +51,4 @@
             "secret": ""
         }
     }
-}
-
+}
```
Comment (tam7t) [inline test/bats/tests/azure/job_templates/kubernetes_windows.json:54]: nit: newline

**Review by nilekhc — APPROVED** [review]
lgtm

**Comment by aramase** [comment]
Intermittent issue pushing image
```bash
docker manifest create --amend k8sprow.azurecr.io/driver:v0.1.0-e2e-4c140b4a  k8sprow.azurecr.io/driver:v0.1.0-e2e-4c140b4a-linux-amd64  k8sprow.azurecr.io/driver:v0.1.0-e2e-4c140b4a-linux-arm64  k8sprow.azurecr.io/driver:v0.1.0-e2e-4c140b4a-windows-1809-amd64  k8sprow.azurecr.io/driver:v0.1.0-e2e-4c140b4a-windows-1903-amd64  k8sprow.azurecr.io/driver:v0.1.0-e2e-4c140b4a-windows-1909-amd64  k8sprow.azurecr.io/driver:v0.1.0-e2e-4c140b4a-windows-2004-amd64
Get https://k8sprow.azurecr.io/v2/driver/manifests/v0.1.0-e2e-4c140b4a-linux-amd64: Get https://k8sprow.azurecr.io/oauth2/token?account=a46d7624-de73-47f7-addb-4d2993a9b10c&scope=repository%3Adriver%3Apush%2Cpull&service=k8sprow.azurecr.io: net/http: TLS handshake timeout
```

/test pull-secrets-store-csi-driver-e2e-windows

**Pull request merged**